
# HG changeset patch
# User David Woodhouse <David.Woodhouse@intel.com>
# Date 1426254219 0
# Node ID a52798da5cfa8b67e2b96b8799af2b00a64a8e1a
# Parent  88b09a22b7c45cee561af2750ea0375419af52bd
Bump farstream02 requirement to 0.2.7

The send-rtcp-mux functionality was added in 0.2.7. Let's try to keep a
cap on the number of variants of different functionality we support, and
make it possible to know that if we're using GStreamer 1.x, we *do* have
the RTCP mux support.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -868,7 +868,7 @@
 	enable_farstream="$enableval", enable_farstream="yes")
 if test "x$enable_farstream" != "xno"; then
 	if test "x$with_gstreamer" == "x1.0"; then
-		PKG_CHECK_MODULES(FARSTREAM, [farstream-0.2 >= 0.2.6], [
+		PKG_CHECK_MODULES(FARSTREAM, [farstream-0.2 >= 0.2.7], [
 			AC_SUBST(FARSTREAM_CFLAGS)
 			AC_SUBST(FARSTREAM_LIBS)
 		], [

